.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+Delete this sentence after you read the "Reporting bugs" paragraph from the
+"guidelines for contributing" link above.
+
+If you are reporting a bug, please try to fill in the following, otherwise
+remove it along with this sentence.
+
+| Question       | Answer
+|----------------|-------------------------------------------------------------
+| Bundle version | composer show --installed|grep sonata-project/admin-bundle
```

Comment:
* Should have back-ticks on command lines.
* Spaces between `|` and commands would be easier to read.